getSelectedAccount (#51)
Concordium/concordium-browser-wallet: packages/browser-wallet-api-helpers/CHANGELOG.md

### Added

-   Expose a JSON-RPC client, using the wallet's current JSON-RPC server.

> -   `getSelectedAccount` method, which allows dApps to get the current account with using `connect`.

Reviewer: ```suggestion
-   `getSelectedAccount` method. This method allows dApps to get the most prioritized account without using `connect`.
```

> -   `getMostRecentlySelectedAccount` method. This method allows dApps to get the most prioritized account without using `connect`.

Reviewer: ```suggestion
-   `getMostRecentlySelectedAccount` method. This method allows dApps to get the most prioritized account without using `connect`. In a future release it will be updated to actually return the most recently selected account.
```
